recipe/meta.yaml

```diff
@@ -23,40 +23,22 @@ requirements:
     - setuptools
     - wheel
   run:
-    - arrow <3.0,>=1.2.0
-    - backoff <4.0,>=2.2.1
-    - beautifulsoup4 <6.0,>=4.8.0
-    - click <10.0
-    - croniter <1.5.0,>=1.3.0
-    - dateutils <2.0
-    - deepdiff <8.0,>=5.7.0
-    - fastapi <2.0,>=0.92.0
-    - fsspec <2025.0,>=2022.5.0
-    - inquirer <5.0,>=2.10.0
-    - jinja2 <5.0
-    - lightning-cloud >=0.5.37
-    - lightning-utilities <2.0,>=0.7.0
-    - numpy <3.0,>=1.17.2
+    - arrow <1.2.4,>=1.2.0
```

Comment:
is it really all, seems to few regarding other is also Apps inside

Reply:
These are the requirements I get from running `grayskull pypi lightning=1.8.6`:

```
requirements:
  host:
    - python >=3.7
    - pip
  run:
    - aiohttp >=3.8.0
    - arrow >=1.2.0
    - beautifulsoup4 >=4.8.0
    - click
    - croniter >=1.3.0,<1.4.0
    - deepdiff >=5.7.0
    - fsspec >2021.06.0
    - inquirer >=2.10.0
    - lightning-cloud >=0.5.12
    - lightning-utilities >=0.3.0,!=0.4.0
    - numpy >=1.17.2
    - packaging >=17.0
    - psutil
    - python >=3.7
    - pytorch >=1.9.0
    - pyyaml >=5.4
    - starsessions >=1.2.1,<2.0
    - tensorboardx >=2.2
    - traitlets >=5.3.0
    - torchmetrics >=0.7.0
    - tqdm >=4.57.0
    - typing-extensions >=4.0.0
```
It does look like there are a few packages missing. Can also cross-reference the requirements in the source tar.gz file at https://pypi.org/project/lightning/1.8.6/#files